### PR TITLE
Clear the keyboard state in the test framework when keyboard is closed.

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
cc @goderbauer

File: packages/flutter_test/lib/src/test_text_input.dart [hunk 1]
```diff
@@ -19,6 +22,19 @@ import 'widget_tester.dart';
 /// * [WidgetTester.showKeyboard], which uses this class to simulate showing the
 ///   popup keyboard and initializing its text.
 class TestTextInput {
+  /// Create a fake keyboard backend.
+  ///
+  /// The [onCleared] argument may be set to be notified of when the handle is
```
Comment (goderbauer) [inline packages/flutter_test/lib/src/test_text_input.dart]: This doc comment is not clear to me. What's the handle? How is it reset?

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter_test/lib/src/test_text_input.dart (hunk 1 above)
Comment (Hixie) [reply packages/flutter_test/lib/src/test_text_input.dart]: changed to "keyboard is dismissed".

**Pull request merged**